docs/API.md

```diff
@@ -293,6 +293,31 @@ By default, the `value` is compared with the current time at the time the functi
 
 **Note:** The reason [`IntlShape`](#Intlshape) has a `now` function is to allow both `<IntlProvider>` and `<FormattedRelative>` components to provide an `initialNow` prop. This allows for the current time to be fixed for things like testing or server-side rendering in an isomorphic/universal React app.
 
+#### `formatRelativeTime`
```

Comment:
We should differentiate how FormatRelativeTime is different from FormatRelative.